subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/generator/FileContentGenerator.groovy

```diff
@@ -29,58 +29,113 @@ class FileContentGenerator {
         if (isRoot && config.subProjects > 0) {
             if (config.compositeBuild) {
                 return """
-                    task clean {
-                        dependsOn gradle.includedBuilds*.task(":clean")
+                    tasks.create("clean") {
```

Comment:
:x: This changes the template to non-idiomatic Groovy and ends-up removing potential overhead in the Groovy translation. We should compare idiomatic Kotlin against idiomatic Groovy.